pkgs/by-name/ya/yarn-berry/fetcher/yarn-berry-fetcher.nix

```diff
@@ -0,0 +1,52 @@
+{
+  lib,
+  rustPlatform,
+  fetchFromGitLab,
+  pkg-config,
+  libzip,
+  zlib,
+  zlib-ng,
+  openssl,
+
+  berryVersion,
+  berryCacheVersion,
+}:
+
+rustPlatform.buildRustPackage (finalAttrs: {
+  pname = "yarn-berry-${toString berryVersion}-fetcher";
+  version = "1.0.0";
+
+  src = fetchFromGitLab {
+    domain = "cyberchaos.dev";
+    owner = "yuka";
+    repo = "yarn-berry-fetcher";
+    tag = "1.0.0";
+    hash = "sha256-iMU/SadzrNv8pZSgp2fBwWVgrgZsnyPRsvs0ugvwyks=";
+  };
+
+  useFetchCargoVendor = true;
+  cargoHash = "sha256-ETFaCu+6Ar7tEeRCbTbesEqx9BdztSvPXB7Dc5KGIx0=";
+
+  YARN_ZIP_SUPPORTED_CACHE_VERSION = berryCacheVersion;
+
+  LIBZIP_SYS_USE_PKG_CONFIG = 1;
```

Comment:
Shouldn't this go into an `env = {}` block?

Reply:
Done